### PR TITLE
Release 1.0.0

### PR DESCRIPTION
**Pull request by richardjgowers**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by richardjgowers** [comment]
@conda-forge-admin, please rerender

**Review by mikemhenry — CHANGES_REQUESTED** [review]
I need to double check pins and re-render

**Comment by mikemhenry** [comment]
I think it is more clear to use this syntax `~=` for specifying versions https://docs.conda.io/projects/conda-build/en/stable/resources/package-spec.html#id3 

```
~=3.1: version 3.1 or later, but not version 4.0 or later.
~=3.1.2: version 3.1.2 or later, but not version 3.2.0 or later.
```

Let me know if I made any mistakes @richardjgowers 

The reason I like the syntax is it makes it easier (once you understand it) to communicate what you mean without having to use 2 inequalities (or one inequality without an upper bound).

 
I know there is a lot of debate on upper bound pins, but as long as we are proactive on keeping track of when new versions come out (and then test them), it _should_ make it better for our users. Most of the pins we have are in the ecosystem we control OR are critical ones to making good science. 

I exclude OpenMM from this (`- openmm ~=8.0`) since it doesn't follow symver

**Comment by mikemhenry** [comment]
This method also catches mistakes like this, where we had an old lomap2 pin:
`- lomap2 >=2.1.0`
BUT installing `lomap2 2.3.1` (which is legal according to our pins) results in this error `ImportError: cannot import name 'LomapAtomMapper' from 'lomap'`

File: recipe/meta.yaml [hunk 1]
```diff
@@ -23,27 +23,27 @@ requirements:
...
     - click
-    - gufe >=1.0.0rc0
-    - lomap2 >=2.1.0
+    - gufe ~=1.0
+    - kartograf ~=1.0
+    - lomap2 ~=3.0
     - networkx
     - numpy
+    - openfe-analysis ~=0.2.0
     - openff-toolkit >=0.13,<0.16
```
Comment (mikemhenry) [inline recipe/meta.yaml:34]: I am going to also double check if the lower limit on this is still true

**Review by mikemhenry — APPROVED** [review]

File: recipe/meta.yaml (hunk 1 above)
Comment (mikemhenry) [reply recipe/meta.yaml:34]: We will fix this with a repometadata patch if needed in the future

**Pull request merged**